File: src/main/java/com/matag/admin/game/score/ScoresResponse.java
package com.matag.admin.game.score;

import java.util.List;

import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Value;

@Value
@AllArgsConstructor
@JsonDeserialize(builder = ScoresResponse.ScoresResponseBuilder.class)
@Builder
public class ScoresResponse {
  List<ScoreResponse> scores;

  @JsonPOJOBuilder(withPrefix = "")
  public static class ScoresResponseBuilder {

  }
}

File: src/main/java/com/matag/admin/game/score/ScoresResponse.kt
package com.matag.admin.game.score;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class ScoresResponse {
  List<ScoreResponse> scores;
}
